python/README.md

```diff
@@ -26,57 +36,26 @@ optional arguments:
   -port PORT, --flightport PORT
                         Dremio flight server port. Defaults to 32010.
   -user USERNAME, --username USERNAME
```

Comment:
We should mention this is not needed when using a PAT.

Reply:
Agreed.